vendor/code.gitea.io/git/Makefile

```diff
@@ -18,7 +18,7 @@ generate:
 
 .PHONY: fmt
 fmt:
-	find . -name "*.go" -type f -not -path "./vendor/*" -not -path "./benchmark/*" | xargs gofmt -s -w
+	find . -name "*.go" -type f ! -path "./vendor/*" ! -path "./benchmark/*" | xargs gofmt -s -w
```

Comment:
Do not make changes to vendor directory. Create PR in github.com/go-gitea/git/ repository for this change